discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown

```diff
@@ -31,35 +26,37 @@ $$$
 
 ## Installing @product@ Dependencies [](id=installing-liferay-dependencies)
 
-@product@ depends on some additional JARs that aren't included with tc Server by
-default. There are even more JARs that you'd find in a @product@ bundle
-that are not required but can be useful. If you don't have a @product@
-bundle, you can download the required JARs from third parties, as
-described below.
+@product depends on some additional JARs that are not included with tc Server by
+@default. There are even more JARs found in a product bundle that are not
+@required but can be useful. If you don't have a product bundle, you can
+@download the required JARs from third parties, as described below.
 
```

Comment:
What happened here with the @ symbols? Editor malfunction? :-) This whole change should be reverted.